CorsixTH/Lua/rooms/general_diag.lua

```diff
@@ -52,47 +52,81 @@ end
 
 function GeneralDiagRoom:commandEnteringPatient(patient)
   local screen, sx, sy = self.world:findObjectNear(patient, "screen")
+  -- Patient walk to screen
```

Comment:
If you want to leave the singular word form in; then really a single line block comment would be nice here to denote they're action constructors.